backend/package.json

```diff
@@ -58,9 +58,11 @@
     "@nestjs/platform-express": "^8.1.1",
     "@nestjs/swagger": "^5.1.4",
     "@nestjs/typeorm": "^8.0.2",
+    "@types/multer": "^1.4.7",
     "bcrypt": "^5.0.1",
     "class-transformer": "^0.4.0",
     "class-validator": "^0.13.1",
+    "cloudinary": "^1.27.1",
```

Comment:
also make sure you do a fresh npm install in the root directory to get these packages